packages/skeleton/src/lib/index.ts

```diff
@@ -10,9 +10,8 @@ export type { ToastSettings } from './utilities/Toast/types.js';
 export type { TableSource } from './components/Table/types.js';
 export type { PaginationSettings } from './components/Paginator/types.js';
 export type { PopupSettings } from './utilities/Popup/types.js';
-
-// This type alias is to identify CSS classes within component props, which enables Tailwind IntelliSense
-export type CssClasses = string;
+export type { Transition, TransitionParams } from './internal/transitions.js';
+export type { CssClasses } from './types.js';
 export type SvelteEvent<E extends Event = Event, T extends EventTarget = Element> = E & { currentTarget: EventTarget & T };
 
```

Comment:
This is unrelated, but could you move these two over to the `types.ts` file and rexport them into `index.ts` (just like how we did it with `CssClasses`)? This PR introduces the new file so we might as well take advantage 😄